subjects/go-reloaded/audit/README.md

```diff
@@ -1,19 +1,19 @@
 #### Functional
 
-###### Has the requirement for the allowed packages been respected? (Reminder for this project: (only [standard packages](https://golang.org/pkg/))
+###### Has the requirement for the allowed packages been respected? (Reminder for this project: (only [standard packages](https://golang.org/pkg/) can be used)
 
-##### In a file called `sample.txt`, place the following text (without the double quotes): "If I make you BREAKFAST IN BED (low, 3) just say thank you instead of: how (cap) did you get in my house (up, 2) ?". Now run the student program with the arguments: `sample.txt result.txt`.
+##### In a file called `sample.txt`, place the following text (without the double quotes): "If I make you BREAKFAST IN BED (low, 3) just say thank you instead of: how (cap) did you get in my house (up, 2) ?". Now run the program with the arguments (without single quotes): `sample.txt result.txt`.
```

Comment:
This is not necessary (without single quotes), as they are not single quotes (')they are (`) and they are used to highlight the words or sentences.
You can see this by opening a preview of the README